openshift/kustomize/core-nlp/base/deploy.yaml

```diff
@@ -30,6 +30,15 @@ spec:
     type: Rolling
   triggers:
     - type: ConfigChange
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+          - corenlp
+        from:
+          kind: ImageStreamTag
+          namespace: 9b301c-tools
+          name: corenlp:dev
```

Comment:
You will need an overlay to handle deploying to TEST and PROD.

Reply:
actually that wasnt something I meant to commit.  It would only be relevant if we introduced a build step for our customer Docker image which I dont see as neccesary.  We dont do it for Kowl or NGINX either.